src/redux/reducers/userReducer.ts

```diff
@@ -36,6 +37,10 @@ export const userReducer = (state: User = emptyUser, action: UserActions | Orcid
         affiliations,
         givenName: action.user.given_name,
         familyName: action.user.family_name,
+        isPublisher: roles.some(role => role === RoleName.PUBLISHER),
+        isAppAdmin: roles.some(role => role === RoleName.APP_ADMIN),
+        isInstitutionAdmin: roles.some(role => role === RoleName.ADMIN),
+        isCurator: roles.some(role => role === RoleName.CURATOR),
```

Comment:
Vi får passe på å be om å få alle roller på våre brukere, siden vi trenger å kunne kjøre uten use-mock

Reply:
Har diskutert det med backend